src/test/cypress/support/pageobjects/multipleChoiceQuiz.ts

```diff
@@ -0,0 +1,19 @@
+import { POST } from '../constants';
+
+export class MultipleChoiceQuiz {
+    getQuizBody() {
+        return cy.get('.mc-question');
+    }
+
+    tickAnswerOption(optionNumber: number) {
+        this.getQuizBody()
+            .get('#answer-option-' + optionNumber + ' > .selection > .ng-fa-icon > .svg-inline--fa')
```

Comment:
Do we really have to click the svg directly? Also the selector is quite unstable because it does search for multiple intermediate elements. Does `.get('#answer-option-' + optionNumber).find('.svg-inline--fa')` not work?

Reply:
had to adapt it a little but i hope you like it more now.